libmscore/style.cpp

```diff
@@ -1382,7 +1381,7 @@ const TextStyle tempoTextStyle {{
       { Sid::tempoFontItalic,                    Pid::FONT_ITALIC            },
       { Sid::tempoFontUnderline,                 Pid::FONT_UNDERLINE         },
       { Sid::tempoAlign,                         Pid::ALIGN                  },
-      { Sid::tempoOffset,                        Pid::OFFSET                 },
+      { Sid::tempoPosAbove,                      Pid::OFFSET                 },
```

Comment:
FWIW I made the same change as part of #4159, except I didn't actually remove tempoOffset from the Sid enum - it jus wouldn't get used any more.  I didn't test but kind of assumed keeping it around would help with compatibility for people who had already created scores in earlier builds and used this style setting.

Reply:
Wouldn't it be better then to "translate" it automatically to the actually used style ID on reading from a file instead of keeping it in the common style IDs table? We could even create a special "styles compatibility table", I guess this is not the first situation when we decide to reconsider some style properties.

Reply:
Probably so.  I hadn't thought quite that far ahead :-)

Reply:
Then I'll update this pull request to include such conversion.